digitalforms-ords-client/pom.xml

```diff
@@ -220,6 +220,23 @@
 			<plugin>
 				<groupId>org.springframework.boot</groupId>
 				<artifactId>spring-boot-maven-plugin</artifactId>
+				<configuration>
+					<skip>true</skip>
+				</configuration>
+			</plugin>
+
+			<plugin>
+			   <artifactId>maven-deploy-plugin</artifactId>
+			   <version>2.8.2</version>
+			   <executions>
+				  <execution>
+					 <id>default-deploy</id>
+					 <phase>deploy</phase>
+					 <goals>
+						<goal>deploy</goal>
+					 </goals>
+				  </execution>
+			   </executions>
 			</plugin>
```

Comment:
I am not sure if this is needed

Reply:
yes, that is right, I added it so in future if we want to deploy our library to nexus it would be easier. I think , it doesn't adversely affect anything